cpu/cc26x0/periph/rfc.c

```diff
@@ -72,13 +58,54 @@ void rfc_irq_disable(void)
     NVIC_EnableIRQ(RF_HW_IRQN);
 }
 
+uint32_t rfc_send_cmd(uint32_t ropAddr)
+{
+    RFC_DBELL->CMDR = ropAddr;
+  /* wait for cmd ack (rop cmd was submitted successfully) */
+  while(RFC_DBELL->RFACKIFG<<31);
+  while(!RFC_DBELL->CMDSTA);
+  if (RFC_DBELL->CMDSTA == 1)
+  {
+    printf("%s: radio operation command acknowledged. \n",__FUNCTION__);
```

Comment:
You've probably noticed that I tend to heavily rely on messages in my code :octocat: I feel like these feedback messages makes developing and debugging a lot easier, especially when you have to interact with a black box like the RF core. IMO providing such debugging tools would be worth the extra cost in terms of memory, especially at the early stages of development... but I guess it is a matter of personal preference :)
I'll put these in a separate commit and keep them for myself then!